docs/src/content/docs/guides/delivering-remote-images.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Delivering Remote Images
+description: Fetching and optimizing remote images with Astro Cloudinary
+head:
+  - tag: title
+    content: Delivering Remote Images - Astro Cloudinary
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+import { CldImage } from '../../../../../astro-cloudinary';
+
+import CodeBlock from '../../../components/CodeBlock.astro';
+import HeaderImage from '../../../components/HeaderImage.astro';
+
+
+Astro Cloudinary provides two powerful methods for fetching, optimizing, and delivering remote images through Cloudinary's transformation pipeline: the Fetch API and Auto Upload Mapping. These features are particularly useful when you want to fetch and optimize external images without explicitly storing them in your Cloudinary account.
+
+## Method 1: Using the Fetch API
+
+The Fetch API allows you to deliver remote images on-the-fly through Cloudinary, applying transformations as needed. This method is ideal for dynamically accessing and transforming images from external sources without storing them in your Cloudinary account.
+
+### Basic Usage
+
+To fetch a remote image, you need to include the full URL of the image in the `src` prop and set `deliveryType="fetch"`.
```

Comment:
can you add the example on the page before the code, similar to what we're doing here?

https://astro.cloudinary.dev/guides/image-overlays
https://github.com/cloudinary-community/astro-cloudinary/blob/main/docs/src/content/docs/guides/image-overlays.mdx?plain=1#L20-L44